pkgs/applications/misc/nwg-look/default.nix

```diff
@@ -0,0 +1,72 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, substituteAll
+, buildGoModule
+, go
+, glib
+, pkg-config
+, cairo
+, gtk3
+, xcur2png
+, libX11
+, zlib
+}:
+
+buildGoModule rec {
+  pname = "nwg-look";
+  version = "0.2.4";
+
+  src = fetchFromGitHub {
+    owner = "nwg-piotr";
+    repo = pname;
+    rev = "v${version}";
+    sha256 = "sha256-wUI58qYkVYgES87HQ4octciDlOJ10oJldbUkFgxRUd4=";
+  };
+
+  vendorHash = "sha256-dev+TV6FITd29EfknwHDNI0gLao7gsC95Mg+3qQs93E=";
+
+  # Replace /usr/ directories with the packages output location
+  # This means it references the correct path
+  patches = [ ./fix-paths.patch ];
+
+  postPatch = ''
+    sed -i 's|@out@|'"''${out}"'|g' main.go tools.go
```

Comment:
How about using substituteInPlace ? Check in nixpkgs for plenty of examples

Reply:
I have had a look at using substituteInPlace and I have been unable to get it to work, I can try again but I doubt it would be succesful

Reply:
How it's going?

Reply:
I still havent been able to get it to work using substituteInPlace

Reply:
This doesn't work ?
```
  substituteInPlace main.go --replace '@out@' $out
```

Reply:
Im just testing!